cypress/support/commands.ts

```diff
@@ -36,6 +37,15 @@ Cypress.Commands.add('createFixtureWorkflow', (fixtureKey, workflowName) => {
 	workflowPage.actions.zoomToFit();
 });
 
+Cypress.Commands.add(
+	'createFixtureWorkflowWithUniqueName',
```

Comment:
what's the advantage of creating a new command vs updating `createFixtureWorkflow` to use a unique name for every workflow?

Reply:
Yeah I think it might be clearer to just one have command